packages/ios_platform_images/pubspec.yaml

```diff
@@ -2,11 +2,11 @@ name: ios_platform_images
 description: A plugin to share images between Flutter and iOS in add-to-app setups.
 repository: https://github.com/flutter/plugins/tree/main/packages/ios_platform_images
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+ios_platform_images%22
-version: 0.2.0+9
+version: 0.3.0
 
 environment:
   sdk: ">=2.14.0 <3.0.0"
-  flutter: ">=2.10.0"
+  flutter: ">=3.3.0"
```

Comment:
@stuartmorgan said upping to 3.3 was okay per trailing stable policies in https://github.com/flutter/flutter/issues/101965#issuecomment-1232771019.

Reply:
Yep, this is also documented at https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#supported-flutter-versions